sig-cli/CONTRIBUTING.md

```diff
@@ -97,35 +90,24 @@ work.
   - Add `kubectl hello-kubernetes -f file`: Print "Hello \<kind of resource\> \<name of resource\>"
   - Add `kubectl hello-kubernetes type/name`: Print "Hello \<kind of resource\> \<name of resource\> \<creation time\>"
 
-**Note:** Consider publishing your command to a fork so a maintainer can look at it.
+Push your branch to your github fork (`git push -f origin
+{branchName}`) so a maintainer can look at it.
 
 ## Your first contribution
 
 ### Adopt an issue
 
-Pick up an [issue] from the backlog by commenting on the issue that you would like to work on it.
-Be sure to mention the author of the issue as well as the SIG cli members `@seans3` and `@mengqiy`.
-
-Using the following comment will make it easier for us to search for issues folks want to have
-assigned to them:
-
-`cc @seans3 @mengqiy I would like to take this`
```

Comment:
I think we are going to want to mention some folks directly so the issue gets past our filters.

Reply:
I see we do this later

Reply:
trying to put in better (not perfect) order